### DISCUSSION TITLE
No ingame sound after OTA update

### DISCUSSION THREAD

**Discussion by MateuszGladala** [OnionUI/Onion#928] [Support]
After upgrading the system via OTA update from Onion-v4.2.0-beta-dev-4c7e2db I don't have ingame sound anymore, just in main menu of the Onion. I have 05.05 firmware installed.

**Comment by MateuszGladala** [comment]
Downgrade to previous beta fixed the issue

**Comment by TeraHz** [comment]
+1 on this. I switched from audioio to oss in retroarch's audio menu and that fixed it for me

**Comment by Aemiii91** [comment]
Should be fixed in latest release. You might need to do manual update.
